<launch>
	<arg name="spin_rate" default="100"/>

	<arg name="input_msg" default="/t300/state"/>
	<arg name="output_msg" default="/carla/ego_vehicle/vehicle_control_cmd_manual"/>

	<node name="pharos_carla_bridge_node" pkg="pharos_carla_bridge" type="pharos_carla_bridge_node" output="screen">
		<param name="spin_rate" value="$(arg spin_rate)"/>

		<param name="input_msg" value="$(arg input_msg)"/>
		<param name="output_msg" value="$(arg output_msg)"/>
	</node>

	<!-- <include file="$(find pharos_pkg)/launch/pharos_launch.launch">
		<arg name="param1" value="$(arg arg1)" />
	</include> -->

</launch>